app/api/authentication_api.rb

```diff
@@ -247,7 +248,7 @@ class AuthenticationApi < Grape::API
     #
     desc 'Sign out'
     delete '/auth/:auth_token' do
-      token = AuthToken.find_by_auth_token(params[:auth_token])
+      token = AuthToken.find_by_encrypted_authentication_token(params[:auth_token])
```

Comment:
Require the username here and then find token via the user - as should be the case everywhere.